tests/jest/unit/core/common/assets/js/api/core/Editor/Panel/search.test.js

```diff
@@ -0,0 +1,76 @@
+import LocalizedValueStore from 'elementor-editor-utils/localized-value-store';
+import UserEvents from 'elementor-editor-utils/user-events';
+
+describe( 'Localization store', () => {
+	function builtKeyInputEvent( originalEventKey, keyCode, originalEventData, targetValue ) {
+		return {
```

Comment:
Try to use a `simulate` method to simulate the keyboard events

Reply:
I don't think simulate can be applied on a js code